platforms/hyperledger-fabric/configuration/roles/create/configtx/templates/configtxProfile.tpl

```diff
@@ -20,8 +20,10 @@ Profiles:
     Consortiums:
       {{channel.consortium}}:
         Organizations:
-{% for org in channel.participants %}
+{% for org in network.organizations %}
```

Comment:
Ideally this should be documented. This would allow one to add consortium members just by keeping those names in the `network.yaml` while creating the channel.

I was thinking about providing this option as an edit to config when required instead.